packages/extension-text-align/src/text-align.ts

```diff
@@ -37,9 +37,13 @@ export const TextAlign = Extension.create<TextAlignOptions>({
         attributes: {
           textAlign: {
             default: this.options.defaultAlignment,
-            renderHTML: attributes => ({
-              style: `text-align: ${attributes.textAlign}`,
-            }),
+            renderHTML: attributes => {
+              if (attributes.textAlign == this.options.defaultAlignment){
+                return {}
+              } else {
+                return { style: `text-align: ${attributes.textAlign}` }
+              }
+            },
```

Comment:
Use `===` for comparison. You don't need `else`. 

```suggestion
            renderHTML: attributes => {
              if (attributes.textAlign === this.options.defaultAlignment) {
                return {};
              }
               
               return { style: `text-align: ${attributes.textAlign}` }
            },
```